contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2355,6 +2355,31 @@ windows_login_contains_invalid_chars(char *input)
 	return false;
 }
 
+/*
+ * Extract the domain name from the orig_loginname
+ */
+char*
+get_windows_domain_name(char* input){
+	char *pos_slash = strchr(input, '\\');
```

Comment:
This returns NULL when there won't be any '\\' char. Do we have any existing testcase `create login xyz from windows;` ?

Reply:
The checks for this are present here: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/670b113350fdd1e8e6be200bf386b79700993b79/contrib/babelfishpg_tsql/src/pl_handler.c#L2708
The error is thrown there and it doesn't reach here.

Reply:
The testcase is present in `BABEL-1442.sql`